### ISSUE TITLE
Column filtering using formulas

### ISSUE THREAD

**Issue by fbreitwieser** [rstudio/DT#506]
It would be great if there was a functionality to filter rows using column formulas. Something like

```
> $2 * 10      # show only rows where the selected column has a value 10 times higher than the second column
```

Is this possible somehow?

**Comment by yihui** [comment]
It is not possible for now, but I think it could be implemented (the question is who and how).

**Comment by fbreitwieser** [comment]
Thanks for the quick reply, @yihui . I made a workaround for now getting the filter string from a textBox and using non-standard evaluation. More or less the following:
```r
filter_string <- "$2 > (10*$1 + $11)"
eval(parse(text=gsub("\\$([0-9]+)", "df[[\\1]]", filter_string))
```
Works well so far. It's probably more intuitive anyway to have such filtering functionality globally instead of for each column.

**Comment by yihui** [comment]
Yeah, I agree once this is officially supported in this package, many users will feel excited!

**Comment by fbreitwieser** [comment]
Somehow related: I tried securing the `eval` with `tryCatch`, but it crashes regardless. Can you suggest a workaround?

**Comment by shrektan** [comment]
Can you provide an example? The following example works for me.

``` r
tryCatch(
  eval(parse(text = "abc")),
  error = function(e) print(e$message)
)
#> [1] "object 'abc' not found"
```

**Comment by fbreitwieser** [comment]
Sorry, seems that I did something wrong there. Please disregard that question
